### PR TITLE
[MIG] mrp_operating_unit: Migration to 14.0

### PR DESCRIPTION
**Pull request by RujiaL**

### CODE REVIEW

**Review by AaronHForgeFlow — APPROVED** [review]
Code review :+1:

**Review by JuanyDForgeflow — APPROVED** [review]
Functional Review is ok

**Comment by AaronHForgeFlow** [comment]
Hi @RujiaL . It seems this is read just because of pre-commit update. Probably running pre-commit should suffice. Can you check?

**Comment by AaronHForgeFlow** [comment]
Hi @RujiaL , This is about getting merge. Will you continue with this PR? only fixing pre-commit is pending.

File: mrp_operating_unit/models/mrp.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+            wh = self.env["stock.warehouse"].search(
+                [("operating_unit_id", "=", self.operating_unit_id.id)], limit=1
+            )
+            picking_type_id = (
+                self.env["stock.picking.type"].search(
+                    [
+                        ("code", "=", "mrp_operation"),
+                        ("company_id", "=", self.company_id.id),
+                        ("warehouse_id", "=", wh.id),
+                    ], limit=1
```
Comment (RujiaL) [inline mrp_operating_unit/models/mrp.py:60]: > Hi @RujiaL , This is about getting merge. Will you continue with this PR? only fixing pre-commit is pending.

@AaronHForgeFlow I just checked where pre-commit failed https://github.com/OCA/operating-unit/runs/5529543156?check_suite_focus=true, I think this pre-commit ran on an out of date commit which now has been fixed

Sorry I've not been working on this anymore, if you can trigger the pre-commit to run this updated file again it should pass

Comment (AaronHForgeFlow) [reply mrp_operating_unit/models/mrp.py:60]: Ok! thanks for letting me know :)

I will supersede this Pull request

**Comment by AaronHForgeFlow** [comment]
Closing as superseded here: https://github.com/OCA/operating-unit/pull/474

**Pull request closed without merging**